aws-lambda/README.md

```diff
@@ -0,0 +1,36 @@
+# What is this?
+
+This is an example [opta](https://github.com/run-x/opta) configuration file to deploy a simple lambda function on AWS that handles API calls from the internet.
+
+# What does this do?
+It deploys a simple [python lambda function](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/python/example_code/lambda/lambda_handler_rest.py) with AWS API Gateway. It also sets up various other resources like VPCs, subnets.
+
+# Steps to deploy
+* Fill in the following required variables in the config file
+  * org_name
+  * account_id
+* Create a zip file for your lambda.
+```bash
+zip my-deployment-package.zip lambda_function.py
+```
+Checkout [AWS docs](https://docs.aws.amazon.com/lambda/latest/dg/python-package.html) on how to package more complex lambdas
+* Run `opta apply`
```

Comment:
may be add a one liner on what will that do (someone might read this page and not have seen opta apply before)